### PR TITLE
BZ1981075: Note about name/namespace for CRs

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1981075

2.0 and next release

https://deploy-preview-174--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#migrating-virtual-machines-cli_forklift

### CODE REVIEW

**Comment by apinnick** [comment]
@mguetta1 Please review

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -17,9 +17,13 @@ You can migrate virtual machines (VMs) from the command line (CLI) by creating t
...
-[NOTE]
+The term _destination_ in the API is the same as _target_ in the web console.
+
+[IMPORTANT]
 ====
-The API uses the term `destination` to refer to the target environment.
+You must specify a name for cluster-scoped CRs.
+
+You must specify both a name and a namespace for namespace-scoped CRs.
 ====
```
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:27]: Looks good

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 2]
```diff
@@ -91,7 +95,7 @@ spec:
     namespace: {namespace}
 EOF
 ----
-<1> Specify the vSphere API end point, for example, `https://<vcenter.host.com>/sdk`.
+<1> Specify the vSphere API end point, for example, `\https://<vcenter.host.com>/sdk`.
 <2> Specify the name of the VMware `Secret` CR.
```
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:99]: '\' before 'https' - should it be there?

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 3]
```diff
@@ -134,21 +138,23 @@ spec:
...
     destination:
-      name: destination-cluster
+      name: <destination_cluster>
       namespace: {namespace}
 EOF
 ----
 <1> Allowed values are `pod` and `multus`.
 <2> Specify the managed object reference of the VMware network.
 <3> Specify the network attachment definition for each additional destination network.
+<4> Specify the network attachment definition namespace.
```
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:157]: Great. Thanks

File: documentation/modules/migrating-virtual-machines-cli.adoc (hunk 2 above)
Comment (apinnick) [reply documentation/modules/migrating-virtual-machines-cli.adoc:99]: Yes. Example URLs need to be escaped so that they do not turn into clickable links.

Comment (apinnick) [reply documentation/modules/migrating-virtual-machines-cli.adoc:99]: Just checked. It looks like the escape character is not working, so I will take it off. I will have to add it when we start publishing downstream docs.

**Pull request merged**